/**
 * get the appropriate failure message from an error object to print
 * in a `not ok` test point when unhandled throws or rejections happen.
 */
export declare const messageFromError: (er: unknown) => string;
//# sourceMappingURL=message-from-error.d.ts.map